.build-tools/builtin-authentication-profiles.yaml

```diff
@@ -103,3 +103,70 @@ azuread:
           - AzurePublicCloud
           - AzureChinaCloud
           - AzureUSGovernmentCloud
+
+gcp:
+  - title: "GCP API call Authentication"
+    description: |
+      Authenticate authenticates API calls with the given service account or refresh token JSON credentials.
+    metadata:
+      - name: private_key_id
```

Comment:
All Dapr components use camelCase for metadata properties, these should not use snake_case